Helper/Quote.php

```diff
@@ -486,4 +494,64 @@ public function validateAllowedProductTypes()
 
         return true;
     }
+
+    /**
+     * Check if for given sku bread checkout is disabled
+     *
+     * @param string $sku
+     * @param string $store
+     * @return bool
+     * @throws \Magento\Framework\Exception\NoSuchEntityException
+     */
+    public function checkDisabledForSku($sku = null, $store = \Magento\Store\Model\ScopeInterface::SCOPE_STORE)
```

Comment:
could the $store parameter just be made a variable, since it's never actually called?

Reply:
I would leave it as is, since that is the way all other methods that fetch config values do the same.